### DISCUSSION TITLE
Adding 'distance_weights' to documentation

### DISCUSSION THREAD

**Discussion by benpage2** [beetbox/beets#5173] [Ideas]
Hi guys, newer beets user here.

While learning to use beets I stumbled across the 'distance_weights:' under the 'match:' configurations in the forums and I was wondering if there was a reason it wasn't added to the documentation? It certainly helped me a lot with some tagging issues.

I'm also a little confused on what some of the variables refer to and how the matches are calculated. Any clarification below would be appreciated. Thanks!

From the 'extra_tags:' configuration description, the documentation states that "beets will use only the artist, album, and track count to query MusicBrainz." Does this change with singletons? I'm also assuming that the weights are only applied to matches returned by the query, right? Does this also mean that 'track_id' and others aren't considered to make queries but only check how close they are? 

    distance_weights:
        source: prefers musicbraiz tagging souce?
        artist: album artist
        album: album title
        media: ?
        mediums: media format? (cd, digital, etc)
        year: album release year
        country: album release country
        label: music lable
        catalognum: ?
        albumdisambig: ?
        album_id: mbrainz album ID
        tracks: # of tracks on album
        missing_tracks: # of imported tracks missing from album
        unmatched_tracks: # of impoted tracks not matched to album
        track_title: track title 
        track_artist: track artist 
        track_index: track index number
        track_length: track length
        track_id: mbrainz Acoustid

**Comment by Ifiht** [comment]
I think documentation would really help for this.

**Comment by arsaboo** [comment]
Documentation would definitely help (I'll try to add some or feel free to create a PR). Meanwhile, refer to [this thread](https://github.com/beetbox/beets/discussions/4297) for details on distance.
